specs/testing/skills/SkillsFunctionalTesting.md

```diff
@@ -200,15 +227,14 @@ Given these elements, the number of test cases for each scenario can be calculat
 
 ### 7. A skill uses team specific APIs
 
-> TODO: Currently not supported but it would involve things like:
->
 > - Retrieve list of channels in a team
 > - Get team info
+> - Get members from a non-team-scoped conversation
 > - Retreive the _paged_ list of uses in a group where the group is large enough to necessitate more than one page.
 
 **Testing matrix**
 
-- Skill: TeamsBot
+- Skill: [TeamsBot](https://github.com/microsoft/BotBuilder-Samples/tree/main/samples/csharp_dotnetcore/57.teams-conversation-bot)
```

Comment:
Move the link to the repo into the descripion for TeamsBot at the bottom of this doc.

Reply:
resolved